src/main/webapp/index.jsp

```diff
@@ -3,12 +3,14 @@
   User: tomgo
   Date: 20/11/2020
   Time: 19:27
-  To change this template use File | Settings | File Templates.
 --%>
 <%@ page contentType="text/html;charset=UTF-8" language="java" %>
 <html>
 <head>
-    <title>Title</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=PT+Sans&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="styles/main.css" />
```

Comment:
The first two lines are for connecting to Google Fonts, to retrieve PT Sans for use on the page.

The third line connects to our local custom CSS document.